src/pages/index.tsx

```diff
@@ -121,7 +135,26 @@ export default function Home() {
         >
           Search
         </button>
-        <p className="mt-2">Label: {topic}</p>
+        <div className="w-full flex items-center justify-center mt-2">
+          <p className="relative p-2 rounded-3xl bg-slate-500 w-full md:w-1/2 text-center">
+            <>
+              {!selectedLabels.length && "Please select a label"}
+              {!!selectedLabels.length &&
+                `${
+                  selectedLabels.length === 1 ? "Label:" : "Labels:"
+                } ${selectedLabels.join(", ")}`}
+            </>
+          </p>
+          {!!selectedLabels.length && (
+            <button
+              type="button"
+              onClick={() => setSelectedLabels([])}
```

Comment:
We want when it clear to select the html label because now when you clear it doesn't display data.
```suggestion
              onClick={() => setSelectedLabels(["html"])}
```

Reply:
I must have missed this. I have set it to "HTML", same as the default state to match the string in the `labelslang` array. It's fixed now.